front/src/common/Map/Layers/MastsLayers.tsx

```diff
@@ -0,0 +1,52 @@
+/**
+ * Shared layer showing the mast for icons on the map
+ */
+import { SymbolLayer } from 'react-map-gl/maplibre';
+import { isNil } from 'lodash';
+import { OmitLayer } from 'types';
+
+/**
+ * Shared layer showing the mast for icons on the map
+ * @param params
+ */
```

Comment:
I think this function is quite clear, no need for the param description
```suggestion
/**
 * Shared layer showing the mast for icons on the map
 */
```